practice/ismounted.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+DIR=${1}
+
+function check_if_mounted {
+	MOUNTED=$(lsblk -o MOUNTPOINT | grep --color "${DIR}")
+
+	if [ -n  "${MOUNTED}" ]
+	then
+		lsblk | grep --color "${DIR}"
```

Comment:
Hogy oldanád meg, hogy ha egy mountpoint alatt van egy másik, akkor csak azt listázza, amelyiket itt épp meghívtad? Így most vissza fogja adni.

Tehát ha ha kiíratod a `/mnt/a/`-t, az ki fogja írni a `/mnt/a/b`-t is.